docs/Makefile

```diff
@@ -28,10 +31,15 @@ clean:
 
 # Catch-all target: route all unknown targets to Sphinx using the new
 # "make mode" option.  $(O) is meant as a shortcut for $(SPHINXOPTS).
-%: Makefile doxygen
+%: Makefile doxygen # gae-java-sdk
```

Comment:
Delete `# gae-java-sdk` if you decide don't impose a new target.

Reply:
BTW I think a seperate target would be better.

Reply:
mvn javadoc::javadoc has to be the last step in doc generation to override the placeholder `reference/gae_java/index`.
A separate target for javadoc will require sphinx build as a pre-requisite.